### ISSUE TITLE
Update failed

### ISSUE THREAD

**Issue by j3r3m067** [naofireblade/homebridge-weather-plus#110]
Hello,
I installed your plugin on my homebridge.
The updated has failed, so I uninstalled and reinstalled the lastest version (3.1.3).
Now I have this message
```
[4/29/2020, 22:23:15] Error: Time zone name must be a string, got Europe/Paris [object]
    at Moment.fn.tz (/usr/local/lib/node_modules/homebridge-weather-plus/node_modules/moment-timezone/moment-timezone.js:629:11)
    at OpenWeatherMapAPI.parseReport (/usr/local/lib/node_modules/homebridge-weather-plus/apis/openweathermap.js:107:51)
    at OpenWeatherMapAPI.<anonymous> (/usr/local/lib/node_modules/homebridge-weather-plus/apis/openweathermap.js:72:10)
    at IncomingMessage.<anonymous> (/usr/local/lib/node_modules/homebridge-weather-plus/node_modules/openweather-apis/index.js:250:18)
    at IncomingMessage.emit (events.js:322:22)
    at endReadableNT (_stream_readable.js:1187:12)
    at processTicksAndRejections (internal/process/task_queues.js:84:21)
[4/29/2020, 22:23:15] TypeError: (name || "").toLowerCase is not a function
    at normalizeName (/usr/local/lib/node_modules/homebridge-weather-plus/node_modules/moment-timezone/moment-timezone.js:399:23)
    at getZone (/usr/local/lib/node_modules/homebridge-weather-plus/node_modules/moment-timezone/moment-timezone.js:421:10)
    at Function.tz (/usr/local/lib/node_modules/homebridge-weather-plus/node_modules/moment-timezone/moment-timezone.js:556:11)
    at OpenWeatherMapAPI.prepareForecasts (/usr/local/lib/node_modules/homebridge-weather-plus/apis/openweathermap.js:177:29)
    at OpenWeatherMapAPI.parseForecasts (/usr/local/lib/node_modules/homebridge-weather-plus/apis/openweathermap.js:128:30)
    at OpenWeatherMapAPI.<anonymous> (/usr/local/lib/node_modules/homebridge-weather-plus/apis/openweathermap.js:86:10)
    at IncomingMessage.<anonymous> (/usr/local/lib/node_modules/homebridge-weather-plus/node_modules/openweather-apis/index.js:250:18)
    at IncomingMessage.emit (events.js:322:22)
    at endReadableNT (_stream_readable.js:1187:12)
    at processTicksAndRejections (internal/process/task_queues.js:84:21)
```

**Comment by paskovitch** [comment]
I have the same error message.

**Comment by naofireblade** [comment]
Thanks for reporting. This is due to an updated dependency. Will fix this in the next few minutes.

**Comment by naofireblade** [comment]
Fixed in v3.1.4

**Comment by paskovitch** [comment]
Thank you!
